client/js/widgets/button.js

```diff
@@ -260,8 +260,13 @@ export class Button extends Widget {
           switch(a.aggregation) {
           case 'first':
             if(collections[a.collection].length)
```

Comment:
I usually add curly brackets to the whole construct as soon as either the `if` or the `else` branch contains more than one instruction.

I have no problem merging it as is though.